webrender/src/frame.rs

```diff
@@ -319,6 +319,31 @@ impl Frame {
                 continue;
             }
 
+            let mut delta:Point2D<f32> = match scroll_location {
+                ScrollLocation::Delta(delta) => delta,
+                ScrollLocation::Start => {
+                    if layer.scrolling.offset.y.round() == 0.0 {
+                        // Nothing to do.
+                        return false;
+                    }
+
+                    layer.scrolling.offset.y = 0.0;
+                    return true;
```

Comment:
So if it's `Start` or `End` you don't want to even proceed to the rest of the function body, even if the scroll happened?